nirc_ehr/resources/web/nirc_ehr/model/sources/NIRCDefault.js

```diff
@@ -65,6 +65,14 @@ EHR.model.DataModelManager.registerMetadata('Default', {
                 width: 150
             },
         },
+        QCState: {
+            getInitialValue: function (v) {
```

Comment:
I thought this was default behavior in Default.js. Is this for the form section qc state?

Reply:
There is a small difference between this configuration and what's in Default.js to prevent QCState from being set to null. If left as null, it later gets set to 'Completed' - this issue occurs with form panels, but not with grid panels.